internal/devhubbot/commands.go

```diff
@@ -173,3 +179,28 @@ func languagesCommandHandler(session *discordgo.Session, message *discordgo.Mess
 
 	_, _ = channelMessageSendF(session, channel.ID, fmt.Sprintf("user %s\n\n%s", username, languages.String()))
 }
+
+func lastRepoCommandHandler(session *discordgo.Session, message *discordgo.MessageCreate, channel *discordgo.Channel, bot *Bot) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	contentParts := strings.Split(strings.TrimSpace(message.Content), " ")
+	if len(contentParts) <= 1 {
+		_, _ = channelMessageSendF(session, channel.ID, "missing github username")
+
+		return
+	}
+
+	username := contentParts[1]
+
+	lastRepo, err := bot.githubService.GetLastRepoByUsername(ctx, username)
+	if err != nil {
+		infra.Logger.Error().Err(err).Msg("github service get the last repo updated by username")
+
+		_, _ = channelMessageSendF(session, channel.ID, fmt.Sprintf("something went wrong retrieving repo for github user %s", username))
```

Comment:
retrieving repo -> retrieving last repo

Reply:
Yes I agree. Thank you for taking the time to examine this code and give good feedback.